pkg/types/ovirt/platform.go

```diff
@@ -0,0 +1,31 @@
+package ovirt
+
+// Platform stores all the global configuration that all
+// machinesets use.
+type Platform struct {
+	URL               string `json:"ovirt_url,omitempty"`
+	Username          string `json:"ovirt_username,omitempty"`
```

Comment:
hmm, secret information in install-config is weird. how does vsphere and baremetal-ipi handle these?

Reply:
I agree and we need to some generic handling for that cause it will be serialized to files, and logs - I don't
think this is something specific to my platform.
Openstack for example use `cloud` which is a yaml file with the password clear text inside.

Reply:
Each platform requires its own authentication method, but no platform has the credentials supplied through the install config. I believe all platforms read the creds from a file on disk, and in some cases might accept an environment variable. Does oVirt have conventions for where connection info is stored. similar to OpenStakck's cloud.yaml? 

If not, perhaps we could take an approach similar to GCP. The installer writes the gcp service account to ~/.gcp/osServiceAccount.json. If that file is not present and the credential information is not stored in standard default locations, then the user is prompted to enter the info.

Reply:
> https://github.com/openshift/installer/blob/d64b4d035d7c0c8aa2c1d4701eb5b4a6270dd23b/pkg/types/vsphere/platform.go#L10

Looks like vSphere accepts username passport as part of the install-config.

But I agree with @patrickdillon that if we could load the creds outside the install-config that would be best. Using a default location and env car to override that location seems good to me.